src/operation.ts

```diff
@@ -0,0 +1,47 @@
+// operation.ts
+
+/*
+ * Copyright (c) 2021-2023 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+
+const operationDate = '2018-01-01T00:00:59.000Z';
```

Comment:
think this should be called START_DATE or something.  also why 59 seconds after midnight?

Reply:
We have logic to filter records based on 'at'   i.e `operation.type === 'create' && operation.createdOn < at ` and this would throw 'currency not found' for at date '2018-01-01T00:00:00.000Z'